### PR TITLE
Changing hostname on the generated ISO

### PR DESCRIPTION
**Pull request by xeon-zolt**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message 
Fixes #https://github.com/fossasia/meilix-generator/issues/79

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: build1.sh [hunk 1]
```diff
@@ -38,6 +38,9 @@ chmod +x meilix-generator.sh
 echo "$(<meilix-generator.sh)" #to test the file
 ./meilix-generator.sh
 rm meilix-generator.sh
+
+#change host name
```
Comment (meets2tarun) [inline build1.sh]: This may solve the #104 ?
or consider including this all https://github.com/fossasia/meilix-generator/issues/79#issuecomment-317121343

Comment (xeon-zolt) [inline build1.sh]: No, it does not solves #104 
I have not tested it for user name change so not including last two lines

**Comment by tabesin** [comment]
The PR naming "this for" is a bit unclear, please improve.

File: build1.sh (hunk 1 above)
Comment (abishekvashok) [inline build1.sh]: It doesnot add hotelos user but an user from the travis tag right?

**Comment by abishekvashok** [comment]
Please change your PR title, commit message etc to make it look more professional

See this: https://chris.beams.io/posts/git-commit/ ,
this: https://gist.github.com/matthewhudson/1475276 ,
and this: https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst

File: build1.sh (hunk 1 above)
Comment (tabesin) [inline build1.sh]: @zeon-zolt As this whole PR is just two lines of code, does this mean you retract the two lines of code and seek a different solution?

**Pull request merged**